tests/test_views.py

```diff
@@ -2848,25 +2848,6 @@ def _make_pointer_only_user_can_see(self, user, project, save=False):
         node = ProjectFactory(creator=user)
         project.add_pointer(node, auth=Auth(user=user), save=save)
 
-    def test_pointer_list_write_contributor_can_remove_private_component_entry(self):
```

Comment:
IIRC, this test existed to prevent an edge case where a pointer was created, but the only people with permission to view it were removed from the project, leading to a state where a pointer could not be deleted from the node. I'm not sure if recent changes have obviated this test, but it might still be important.